pkg/controllers/management/rkeworkerupgrader/node.go

```diff
@@ -185,3 +186,48 @@ func workerOnly(roles []string) bool {
 	}
 	return worker
 }
+
+func CanUpgrade(nodes []*v3.Node, upgradeStrategy *v3.NodeUpgradeStrategy) error {
```

Comment:
nit (doesnt block merging): why not have this function in the `pkg/api/store/clsuter` package? that's the only place it is used.

Reply:
I didn't want to export `calculateMaxUnavailable` and include other node imports in store, since I already had them in here, moved it